tests/behaviour/steps/cli.py

```diff
@@ -80,6 +85,35 @@ def step_impl(context):
     assert log_file.exists(), f"Files in log directory: {list(log_dir.rglob('**/*'))}"
 
 
+@then("the command line output should display the log message")
+def step_impl(context):
+    (status_code, log_message) = context.csvcubed_cli_result
+    expected_log_message = context.text.strip()
+    assert expected_log_message in log_message, log_message
+
+
+@then("the command line output should not display the log message")
+def step_impl(context):
+    (status_code, log_message) = context.csvcubed_cli_result
+    expected_log_message = context.text.strip()
+    assert expected_log_message not in log_message, log_message
+
+
+@then(
+    'a valid code-list is created and serialised to CSVW from the config file "{config_file}"'
+)
+def step_imp(context, config_file: Path):
+    _temp_test_cases_dir = get_context_temp_dir_path(context)
+    config_file_path = _test_case_dir / config_file
```

Comment:
You should really be copying the file into the temporary directory before running it like this. If the  `_build_code_list` function were to alter the input file in anyway then our tests would work once then fail after that.

Reply:
So this solution seems to fulfill the request and works: 
 
`  shutil.copy((_test_case_dir / config_file), _temp_test_cases_dir)

    config_file_path = _temp_test_cases_dir / config_file`